test/test_hybrid_system.jl

```diff
@@ -64,18 +64,22 @@ end
     @test electric_load !== nothing
     @test thermal_unit !== nothing
 
-    ts = collect(get_time_series_multiple(h_sys; type = TimeSeriesData))
-    @test length(ts) == num_time_series
-    @test Set((get_name(x) for x in ts)) == expected_time_series_names
+    sts = collect(get_time_series_multiple(h_sys; type = SingleTimeSeries))
+    forecasts =
+        collect(get_time_series_multiple(h_sys; type = DeterministicSingleTimeSeries))
+    @test length(sts) == 2
+    @test length(forecasts) == 2
+    @test issubset((get_name(x) for x in sts), expected_time_series_names)
+    @test issubset((get_name(x) for x in forecasts), expected_time_series_names)
 
     @test get_time_series(SingleTimeSeries, electric_load, "max_active_power") isa
           SingleTimeSeries
-    @test get_time_series(AbstractDeterministic, electric_load, "max_active_power") isa
+    @test get_time_series(Deterministic, electric_load, "max_active_power") isa
```

Comment:
Just calling out the loss of this functionality. Like I said in the IS PR, I'm open to restoring it but would prefer not to. I am hoping that users can view what's available (show_time_series, list_time_series_info) and then ask for exactly what they want.

Reply:
Let's restore later along with the restoration in IS we discussed